.github/workflows/nightly-unit-tests.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
       max-parallel: 4
       matrix:
         # Run acceptance tests all supported combinations of Ruby and ActiveRecord.
-        ruby: ["3.1", "3.2", "3.3"]
+        ruby: ["3.1", "3.2", "3.3", "3.4"]
         ar: ["~> 7.0.0", "~> 7.1.0", "~> 7.2.0", "~> 8.0.0"]
         # Exclude combinations that are not supported.
         exclude:
```

Comment:
You also need to add excludes for 3.4 with 7.x here. This does not show up in the PR build, as these are only executed during nightly builds.

Reply:
done